### ISSUE TITLE
Electron start with an error alert relate this package.

### ISSUE THREAD

**Issue by elantion** [moxystudio/node-proper-lockfile#87]
I am using electron with ipfs. When start running ipfs, after 20 seconds, it pop up an error alert.

```
Uncaught Exception:
Error: Unable to update lock within the stale threshold
    at options.fs.stat (/Users/yinjames/Projects/lc-ipfs/node_modules/proper-lockfile/lib/lockfile.js:121:25)
    at /Users/yinjames/Projects/lc-ipfs/node_modules/graceful-fs/polyfills.js:285:20
    at FSReqWrap.oncomplete (fs.js:155:5)
```
It seems like an error with node-proper-lockfile. So I post an issue here. You can reproduce this error by using [ipfs electron example](https://github.com/ipfs/js-ipfs/tree/master/examples/run-in-electron).
Relate post here: https://github.com/ipfs/js-ipfs/pull/1962

**Comment by satazor** [comment]
This is probably a duplicate of #82

**Comment by elantion** [comment]
I am not sure it is duplicate or not. I am using a different OS, Mac Mojave with APFS. I will test it after pull request merge. Thanks mate.

**Comment by elantion** [comment]
Tested OK with me. And do you forget to remove the log?

```
precision s 1554254656460 1554254656000
precision s 1554254666000 1554254666000
precision s 1554254676000 1554254676000
precision s 1554254686000 1554254686000
precision s 1554254696000 1554254696000
precision s 1554254706000 1554254706000
precision s 1554254716000 1554254716000
precision s 1554254726000 1554254726000
precision s 1554254736000 1554254736000
precision s 1554254746000 1554254746000
precision s 1554254756000 1554254756000
precision s 1554254766000 1554254766000
precision s 1554254776000 1554254776000
precision s 1554254786000 1554254786000
precision s 1554254796000 1554254796000
precision s 1554254806000 1554254806000
precision s 1554254816000 1554254816000
precision s 1554254826000 1554254826000
precision s 1554254836000 1554254836000
precision s 1554254846000 1554254846000
precision s 1554254856000 1554254856000
precision s 1554254866000 1554254866000
```

**Comment by satazor** [comment]
I left it there so that @pimlie could debug too.

**Comment by satazor** [comment]
This should have been fixed in #88
